nxrm-aws-resiliency/values.yaml

```diff
@@ -25,6 +25,7 @@ deployment:
     env:
       nexusDBName: nexus
       nexusDBPort: 3306
+      install4jAddVmParams: "-Xms2703m -Xmx2703m -XX:MaxDirectMemorySize=2703m"
```

Comment:
Is env the right place for these? I wasn't entirely sure.

Reply:
Seems reasonable  i.e. we can think of the JVM settings as environment's settings